charts/open-notificaties/values.yaml

```diff
@@ -21,16 +21,16 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
-  # fsGroup: 2000
+podSecurityContext:
+  runAsUser: 1000
 
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+securityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+  runAsUser: 1000
```

Comment:
I think this is not "required". When USER is specified in the Dockerfile, you can leave this empty.

Reply:
See https://github.com/open-zaak/charts/pull/18#discussion_r757619170